### PR TITLE
Adding preview to CI

### PR DESCRIPTION
**Pull request by ecamellini**
Pre-requisites:
- [x] merging https://github.com/buildo/terraform/pull/236
- [x] applying terraform changes
- [ ] deploying new `github-workflow-pal` from `master` (not really required for the merge --> just check http://react-components.preview.our.buildo.io/adding-preview.html)
- [x] testing preview on this PR (pipeline already applied)

### CODE REVIEW

File: ci/tasks/build-preview.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+set -x
+
+cd react-components
+
+yarn install --no-progress
+
+ yarn build-styleguidist
```
Comment (gabro) [inline ci/tasks/build-preview.sh]: extra space?

**Comment by ecamellini** [comment]
It doesn't work, but it should 🤷‍♂️

**Comment by ecamellini** [comment]
Found the problem, let's see

**Comment by ecamellini** [comment]
http://react-components.preview.our.buildo.io/adding-preview.html it works! But...since this build does not produce a bundle with a different hash every time => all files are overwritten. This means that only the latest build will be shown in preview, and can be misleading.

Maybe it's not a problem? There are not many PRs in parallel on BRC.

See:
http://react-components.preview.our.buildo.io/adding-preview.html

**Comment by ecamellini** [comment]
@gabro now it works: I managed to produce a bundle and I'm uploading to the bucket only index file and bundle files:

image

content of the build folder:

image

**Pull request merged**